ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountStatsMySqlStore.java

```diff
@@ -200,16 +202,22 @@ public synchronized void storeAccountStats(StatsWrapper statsWrapper) throws SQL
       StatsSnapshot prevAccountStatsSnapshot =
           prevPartitionMap.getOrDefault(partitionIdKey, new StatsSnapshot((long) 0, new HashMap<>()));
       short partitionId = Utils.partitionIdFromStatsPartitionKey(partitionIdKey);
```

Comment:
minor: partition id is actually `long` type, see `AmbryPartition`, maybe make them consistent ? (I see both `statsPartitionKey` and `partitionIdFromStatsPartitionKey` are using short type)

Reply:
Change it to int.